serverless.yml

```diff
@@ -93,3 +93,30 @@ functions:
           Action: ['dynamodb:GetItem'],
           Resource: 'arn:aws:dynamodb:us-east-2:547511976516:table/user-settings',
         }
+  postEventHandlersTcrEmails:
```

Comment:
"TCR"

Reply:
T2CR

Reply:
I will change it to just "TCR" as this will probably be used to handle notifications from both the Token TCR and the Badge TCR..

Reply:
TCR is token curated registry, it's way too broad. Badges are obviously part of the T2CR.

Reply:
What I mean is that there are 2 TCR applications: one is the T²CR and the other is the Badge TCR. 
If it is going to handle notifications from both contracts it is more accurate to just call it TCR.

Reply:
I believe it is probably better to split this into two files, so for this I will use T2CR as you suggested.

Reply:
TCR stands for Token Curated Registry. That would encompass every curated list. The T2CR is the token token curated list. The fact that it is a composition of TCR contracts is an implementation detail. Unless you want to handle notifications for badges and token listings differently, it is fine to refer to the whole as T2CR as that is the name of the application.

Reply:
It is not a composition, the two are independent and exist without any references to each other at all. So if I were to have single file with handlers for two different applications, it would be incorrect to name it t2cr.

Reply:
They are part of the same application: https://kleros-t2cr.netlify.com

You set notifications for an application, not a smart contract.

Reply:
Perhaps the definition of application is too gray, but I believe that the badge TCR is an application and the token TCR is another, even if there is one tool that makes interacting with either of them more pleasant.

I've pushed code for handling events from the badge tcr, which is in another endpoint. I believe this is better than concentrating all events into a single endpoint and then having to write code to switch between event handlers (specially considering we will probably add more badges in the future).

Reply:
Yes I agree that two separate handlers are better. What I was saying was that if you are going to just have one, "T2CR" is better than "TCR", because "TCR" would include all possible curated lists (doges, listia, etc), not just badges and tokens.

Reply:
That would not be incorrect, but the the 2 handler approach is better.